### PR TITLE
NewHaven display support for MAX78002 EvKit. Replaced sprintf() with snprintf() function

### PR DESCRIPTION
**Pull request by Jake-Carter**
@vicloginov PR #186 transferred to this one because of some issues with the private fork

This PR depends on a MiscDrivers update:
* https://github.com/Analog-Devices-MSDK/Libraries-miscDrivers/pull/18

### CODE REVIEW

File: Libraries/Boards/MAX78002/EvKit_V1/Include/board.h [hunk 1]
```diff
@@ -37,6 +37,13 @@
  */
 
 #include <stdio.h>
+#ifdef TFT_ADAFRUIT
```
Comment (Jake-Carter) [inline Libraries/Boards/MAX78002/EvKit_V1/Include/board.h:40]: @vicloginov FYI I moved these to board.h

File: Libraries/Boards/MAX78002/EvKit_V1/board.mk [hunk 2]
```diff
@@ -35,13 +35,25 @@ ifeq "$(BOARD_DIR)" ""
 $(error BOARD_DIR must be set)
 endif
 
+TFT?=ADAFRUIT
```
Comment (Jake-Carter) [inline Libraries/Boards/MAX78002/EvKit_V1/board.mk:38]: Used `?=` so it can be overridden from project.mk

**Review by Jake-Carter — COMMENTED** [review]
@vicloginov Looks good, builds cleanly with `make TFT=NEWHAVEN` against https://github.com/Analog-Devices-MSDK/Libraries-miscDrivers/pull/9

Approved, but I will hold off on merging until we get adapter boards in Dallas to test

File: Libraries/Boards/MAX78002/EvKit_V1/Include/board.h (hunk 1 above)
Comment (vicloginov) [reply Libraries/Boards/MAX78002/EvKit_V1/Include/board.h:40]: No problem

File: Libraries/Boards/MAX78002/EvKit_V1/board.mk (hunk 2 above)
Comment (vicloginov) [reply Libraries/Boards/MAX78002/EvKit_V1/board.mk:38]: Agree

**Review by vicloginov — COMMENTED** [review]
Looks good

File: Examples/MAX78000/CNN/asl_demo/main.c [hunk 3]
```diff
@@ -468,14 +468,15 @@ uint8_t check_inference(q15_t *ml_soft, int32_t *ml_data, int16_t *out_class, do
...
+            //snprintf(buff, sizeof(buff), "__________________________ ");
             //TFT_Print(buff, 1, 50, urw_gothic_12_white_bg_grey);
-            //sprintf(buff, "Top classes:");
+            //snprintf(buff, sizeof(buff), "Top classes:");
             //TFT_Print(buff, 1, 80, urw_gothic_13_white_bg_grey);
         } else {
-            //sprintf(buff, "%s (%0.1f%%)", classes[max_index],
+            //snprintf(buff, sizeof(buff), "%s (%0.1f%%)", classes[max_index],
             //         (double) 100.0 * max / 32768.0);
             //TFT_Print(buff, 20, 80 + 20 * top, urw_gothic_12_white_bg_grey);
```
Comment (sihyung-maxim) [inline Examples/MAX78000/CNN/asl_demo/main.c:481]: Is the commented code something that's still in development and requires more testing? Or should it be deleted? This else block does nothing.

Comment (vicloginov) [reply Examples/MAX78000/CNN/asl_demo/main.c:481]: It's ok to delete commented lines

**Review by sihyung-maxim — APPROVED** [review]

**Review by lorne-maxim — APPROVED** [review]

**Review by lorne-maxim — APPROVED** [review]

**Pull request merged**